docs/_generate/item.html

```diff
@@ -49,4 +49,26 @@
 {% endhighlight %}
   </div>
   <% } %>
+  <% if (prop['sdk-support']) {%>
+  <table>
+    <caption>SDK Version Requirements</caption>
+    <thead>
+    <tr><th></th><th>Mapbox GL JS</th><th>Mapbox iOS SDK</th><th>Mapbox Android SDK</th></tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td>Base support</td>
+      <td><%- prop['sdk-support'].basic.js ? '>= ' + prop['sdk-support'].basic.js : 'Not supported' %></td>
+      <td><%- prop['sdk-support'].basic.ios ? '>= ' + prop['sdk-support'].basic.ios : 'Not supported' %></td>
+      <td><%- prop['sdk-support'].basic.android ? '>= ' + prop['sdk-support'].basic.android : 'Not supported' %></td>
+    </tr>
+    <tr>
+      <td>Data-driven styling</td>
+      <td><%- (prop['sdk-support']['property-function'] || {}).js ? '>= ' + (prop['sdk-support']['property-function'] || {}).js : 'Not supported' %></td>
+      <td><%- (prop['sdk-support']['property-function'] || {}).ios ? '>= ' + (prop['sdk-support']['property-function'] || {}).ios : 'Not supported' %></td>
+      <td><%- (prop['sdk-support']['property-function'] || {}).android ? '>= ' + (prop['sdk-support']['property-function'] || {}).android : 'Not supported' %></td>
+    </tr>
```

Comment:
I definitely want to keep it extensible if we decide there are other relevant rows. However I think we want to keep the key in the JSON decoupled from the terminology in the docs. I can see the key in the JSON becoming relied on by other software (e.g. Studio), so we'd want to keep it stable, while at the same time we might want to change the terminology presented to the end user. So here for example I used `property-function` to match other parts of the JSON, but branded it "Data-driven styling" in the docs. And if this mapping expands beyond two rows, we'd certainly want to make the template more generic.